test/jobs/split_pdf_job_test.rb

```diff
@@ -56,12 +56,6 @@ class SplitPDFJobTest < ActiveJob::TestCase
       should 'have 1 pdf in incomplete directory' do
         assert_equal Dir.entries(@exam_template.base_path + '/incomplete/21/').sort, %w[. .. 1.pdf].sort
       end
-
-      should 'have 1 pdf in error directory' do
```

Comment:
Deleting tests is often a bad sign. If there are changes that invalidate existing tests, you should work to modify them properly.